### ISSUE TITLE
Search bar result count always displays "no results"

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#726]
## Description

The search bar result count just displays "no results" for every search regardless of how many results actually exist.

## Reproduction

1. Visit https://wordpress.org/openverse/search/?q=birds and check out the search bar. It will say "no results"

## Expectation

It should say the number of results following the rules we originally had for that count.

## Additional context

Marking this as a11y and critical as screen reader users could easily be confused by this and thing their search yielded no results.

## Resolution

- [x] 🙋 I would be interested in resolving this bug.

**Comment by krysal** [comment]
I am seeing the correct text:

![CleanShot 2022-01-26 at 10 59 12@2x](https://user-images.githubusercontent.com/9145885/151203056-398606c4-f446-42fa-b19a-afeb5a9febab.png)

I have seen this problem before though, or it just shows the number of an old search result, but I thought it was due to a bad connection. It can be difficult to reproduce.

**Comment by sarayourfriend** [comment]
Interesting. The other thing I'm noticing is that on the initial render it will show "no results" and then switch to "over 10k". Given we already know the result count during SSR I'm not sure why the first render says "no results" 🤔 Maybe there's a way where it gets stuck on that initial render somehow?

**Comment by zackkrida** [comment]
I'm not able to reproduce scenarios where the incorrect count displays in the header, but I _do_ see the flash of 'no results' on initial render and that would be great to resolve.

**Comment by obulat** [comment]
I cannot figure out how to watch the store for changes. The SSR is supposed to be set up so that whenever the result count changes, the search bar result also updates, but it doesn't work. I've tried using a watcher in the setup function (for `store.state.media.results` or for `store.getters['media/resultsCount']`), but they are never called :(

I hope that when we update to Pinia, this problem will be solved.

**Comment by krysal** [comment]
Lately, I notice more that it stays in the "Loading..." state (the **main** branch, not production). Perhaps this is a silly question but @obulat making the `searchStatus` of the `VHeader` a computed property didn't work for some reason? Locally looks like it solves the issue for me.

**Comment by zackkrida** [comment]
I'm seeing the same behavior locally

**Comment by obulat** [comment]
I have noticed the "Loading..." state, too, @krysal , and I'm not sure what the reason for it is.
